### PR TITLE
Update command-flag-element.md

### PR DESCRIPTION
**Pull request by ryanmolden**
Add a couple undocumented but public command flags for use in VSCT files.

### CODE REVIEW

**Comment by ryanmolden** [comment]
#sign-off

**Comment by Court72** [comment]
@maiak

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by musliadi426** [comment]
thank you and nice to meet each other 
👌🌹

**Review by musliadi426 — CHANGES_REQUESTED** [review]
[Uploading FTP.musliadi.Net…]()
Mikrosoft konvertion

File: docs/extensibility/command-flag-element.md [hunk 1]
```diff
@@ -61,6 +62,7 @@ Modifies its parent element.
 |TextMenuCtrlUseMenu|Use the `MenuText` field on menu controllers. The default field is `ButtonText`.<br /><br /> Valid for: `Button`|
 |TextMenuUseButton|Use the `ButtonText` field for menus. The default field is `MenuText` if it is specified.<br /><br /> Valid for: `Button`|
 |TextOnly|Show only text on a toolbar or a menu but no icon even if the icon is specified.<br /><br /> Valid for: `Button`|
+|TogglePatternAvailable|Indicates your button supports being checked/latched (i.e. may return OLEMCDF_LATCHED or set the Checked property on OleMenuCommand). This allows the automation layer to expose the toggle pattern so screen readers will read the checked/unchecked state along with your button text.<br /><br /> Valid for: `Button`|
```
Comment (ghogen) [inline docs/extensibility/command-flag-element.md:65]:
```suggestion
|TogglePatternAvailable|Indicates your button supports being checked/latched (that is, may return OLEMCDF_LATCHED or set the Checked property on OleMenuCommand). This allows the automation layer to expose the toggle pattern so screen readers will read the checked/unchecked state along with your button text.<br /><br /> Valid for: `Button`|
```

**Review by BertanAygun — APPROVED** [review]

**Review by RyanToth3 — APPROVED** [review]

**Pull request merged**